core/plugins/org.fusesource.ide.camel.model.service.core/src/org/fusesource/ide/camel/model/service/core/catalog/CamelModelFactory.java

```diff
@@ -46,10 +49,12 @@
 		camelVersionToFuseBOMMapping.put("2.15.1.redhat-621117", "6.2.1.redhat-117");
 		camelVersionToFuseBOMMapping.put("2.17.0.redhat-630187", "6.3.0.redhat-187");
 		camelVersionToFuseBOMMapping.put("2.17.3",               "6.3.0.redhat-187");
-		camelVersionToFuseBOMMapping.put("2.18.1",               "6.3.0.redhat-187");
```

Comment:
why is the FIS version not included here? There are non-FIS users waiting for the feb release who want to use the newer camel version.